apps/atqr/web/src/pages/termsAndCondition/termsAndCondition.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
```

Comment:
Use pug files.

Reply:
@albnunes @yfernandes I'm taking care of this one